bin/test-workflow/7_app_deploy.sh

```diff
@@ -29,15 +29,27 @@ pushd ../../helm/conjur-app-deploy > /dev/null
     --set app-secretless-broker.conjur.authnLogin=$CONJUR_AUTHN_LOGIN_PREFIX/test-app-secretless-broker \
     --set app-secretless-broker.conjur.authnConfigMap.name=conjur-authn-configmap-secretless \
     --set app-secretless-broker.app.platform=$PLATFORM"
+  secrets_provider_standalone_options="--set app-secrets-provider-standalone.enabled=true \
+    --set app-secrets-provider-standalone.secrets-provider.environment.conjur.authnLogin="$CONJUR_AUTHN_LOGIN_PREFIX/test-app-secrets-provider-standalone" \
+    --set app-secrets-provider-standalone.app.image.tag="$TEST_APP_TAG" \
+    --set app-secrets-provider-standalone.app.image.repository="$TEST_APP_REPO" \
+    --set app-secrets-provider-standalone.app.platform=$PLATFORM"
 
   declare -A app_options
   app_options[summon-sidecar]="$summon_sidecar_options"
   app_options[secretless-broker]="$secretless_broker_options"
+  app_options[secrets-provider-standalone]="$secrets_provider_standalone_options"
 
   # restore array of apps to install
   declare -a install_apps=($(split_on_comma_delimiter $INSTALL_APPS))
   options_string=""
   for app in "${install_apps[@]}"; do
+    # If application that uses Secrets Provider in standalone mode is enabled,
+    # make sure that the Secrets Provider Helm chart has been downloaded as a
+    # dependency for that application's subchart.
+    if [ "$app" = "secrets-provider-standalone" ]; then
+      helm dependency update charts/app-secrets-provider-standalone
+    fi
```

Comment:
Oh, another bit - there are a couple of files that I think are generated by this command:
`helm/conjur-app-deploy/charts/app-secrets-provider-standalone/Chart.lock` and
`helm/conjur-app-deploy/charts/app-secrets-provider-standalone/charts/secrets-provider-1.1.5.tgz`
I think at least the TGZ file should be added to `.gitignore` - not sure about `Chart.lock`

Reply:
Another good catch, thanks! Ignore entries are needed for both files.